Source/m_menu.h

```diff
@@ -117,17 +117,18 @@ extern int warning_about_changes, print_warning_about_changes;
 #define S_DISABLE 0x1000000 // Disable item
 #define S_COSMETIC 0x2000000 // Don't warn about change
 #define S_THERMO  0x4000000 // Mini-thermo
+#define S_NAME    0x8000000 // Player name
```

Comment:
Good idea, better than reusing `S_CHAT`. Can we get rid of `S_FILE`?

Reply:
> Can we get rid of `S_FILE`?

Yes, and we can remove WAD/DEH loader settings from m_misc.c

Reply:
Let's do it.